### PR TITLE
This is my assignment2

### PR DESCRIPTION
**Pull request by DyapaHarishwarReddy**

### CODE REVIEW

**Comment by manchalavaishnavi** [comment]
the layout should be full width not scrollable. all the tags are used properly

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "quixote": "^1.0.1"
   },
   "scripts": {
-    "dev": "./jake karma & ./watch loose=true"
```
Comment (Bhanuprasadhub) [inline package.json:10]: remove . & / from package.json

File: src/Assignment-2/assignment2.css [hunk 2]
```diff
@@ -1,7 +1,77 @@
...
+    float:right;
+    width:80%;
+}
+aside{
+    width:15%;
+}
 
-.container, .content {
-width: 100%;
-height: 100%;
```
Comment (Bhanuprasadhub) [inline src/Assignment-2/assignment2.css:6]: add "{" in css line 58

File: src/Assignment-2/assignment2.html [hunk 3]
```diff
@@ -12,8 +12,68 @@
 	<div id="container">
 
 	<!-- Start your code from here -->
-	
-
```
Comment (Bhanuprasadhub) [inline src/Assignment-2/assignment2.html:16]: check spacing and start the code from top i.e, from line 15 in html

**Review by Bhanuprasadhub — COMMENTED** [review]
check css and html once again

**Pull request merged**